rhobs/alerting/data_plane/prometheus.gitops_routes_alerts.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-routes-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_routess_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsRoutesHealthErrors
+      expr: |
+        (
+          sum(openshift_route_status{ status="True", namespace="gitops-service-argocd"})
```

Comment:
What's the reason for using `sum` here?

Reply:
Here we get series for multiple routes, to handle that I used `SUM()`.

Reply:
Yes, as of now there is only one route created in `gitops-service-argocd` namespace, so there is only one series.